gateway/api_test.go

```diff
@@ -2193,76 +2189,6 @@ func TestOAS(t *testing.T) {
 				})
 			})
 
-			t.Run("with oas and gateway tags enabled", func(t *testing.T) {
```

Comment:
I'd like to keep this one but move it accordingly. It tests acceptance of inputs and outputs via API and provides a contract of implementation.